infra/Makefile.toml

```diff
@@ -5,17 +5,34 @@ extend = [
   { path = "scripts/ethapi.toml" },
   { path = "scripts/node.toml" },
   { path = "scripts/testnet.toml" },
-  { path = "scripts/testnode.toml" }
+  { path = "scripts/testnode.toml" },
+  { path = "scripts/subnet.toml" }
 ]
 
 [config]
 default_to_workspace = false
 
 [env]
+# General network-specific parameters
 CHAIN_NAME = { value = "root", condition = { env_not_set = ["CHAIN_NAME"] }}
 BALANCE = { value = "1000", condition = { env_not_set = ["BALANCE"] }}
 BASE_FEE = { value = "1000", condition = { env_not_set = ["BASE_FEE"] }}
 TIMESTAMP = { value = "1680101412", condition = { env_not_set = ["TIMESTAMP"] }}
+POWER_SCALE = { value = "3", condition = { env_not_set = ["POWER_SCALE"] }}
+CMT_HOST_PORT = { value = "26657", condition = { env_not_set = ["CMT_HOST_PORT"] }}
+ETHAPI_HOST_PORT = { value = "8545", condition = { env_not_set = ["ETHAPI_HOST_PORT"] }}
+
+# IPC subnet related parameters
+# Use calibration as default value
+NODE_NAME = { value = "ipc-node", condition = { env_not_set = ["NODE_NAME"] } }
+PARENT_ENDPOINT = { value = "https://api.calibration.node.glif.io/rpc/v1", condition = { env_not_set = ["PARENT_ENDPOINT"] } }
+PARENT_GATEWAY = { value = "0x56948d2CFaa2EF355B8C08Ac925202db212146D1", condition = { env_not_set = ["PARENT_GATEWAY"] } }
+PARENT_REGISTRY = { value = "0x6A4884D2B6A597792dC68014D4B7C117cca5668e", condition = { env_not_set = ["PARENT_REGISTRY"] } }
+NETWORK_TYPE = { value = "test", condition = { env_not_set = ["NETWORK_TYPE"] } }
+VALIDATOR_PRIV_KEY = { value = "~/.ipc/priv.key", condition = { env_not_set = ["VALIDATOR_PRIV_KEY"] } }
```

Comment:
There already exists a setting for the [validator_key](https://github.com/consensus-shipyard/fendermint/blob/b1dc6c1218331b39553c5990db2f6f8e6513f5ff/fendermint/app/settings/src/lib.rs#L110) and it is [relative to the home directory](https://github.com/consensus-shipyard/fendermint/blob/b1dc6c1218331b39553c5990db2f6f8e6513f5ff/fendermint/app/settings/src/lib.rs#L70), which [by default](https://github.com/consensus-shipyard/fendermint/blob/b1dc6c1218331b39553c5990db2f6f8e6513f5ff/fendermint/app/options/src/lib.rs#L73) is `~/.fendermint`. 

Why should we suddenly have a separate copy under an `~/.ipc` directory too?

Reply:
Are all infra scripts using this different home direcory?

Reply:
So the goal of this variable is to pick up the export of a private key from the `ipc-cli` (so initially it can be persisted anywhere and not necessarily inside the `fendermint` path or with a Fendermint format). [The scripts](https://github.com/consensus-shipyard/fendermint/pull/320/files#diff-fb7d01e502bec77131b22c42a51ca3fa92060ae09be7538f10bf29584a6f746eR55-R56) will pick up this secret key, convert it into a Fendermint private key, and put it into the right location. 

This is why I decided to use a different env variable for this, as this is an external key that is picked up and converted into a Fendermint format to use it for the validator.